libnavigation-core/api/current.txt

```diff
@@ -184,6 +185,50 @@ package com.mapbox.navigation.core.directions.session {
 
 }
 
+package com.mapbox.navigation.core.fasterroute {
+
+  @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public final class FasterRouteOptions {
+    method public double getMaxGeometrySimilarityToRejectedAlternatives();
+    property public final double maxGeometrySimilarityToRejectedAlternatives;
+  }
+
+  public static final class FasterRouteOptions.Builder {
+    ctor public FasterRouteOptions.Builder();
+    method public com.mapbox.navigation.core.fasterroute.FasterRouteOptions build();
+    method public void maxGeometrySimilarityToRejectedAlternatives(double value);
+  }
+
+  @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public final class FasterRoutesTracker {
+    method @UiThread public void acceptFasterRoute(com.mapbox.navigation.core.fasterroute.NewFasterRoute newFasterRoute);
+    method @UiThread public void declineFasterRoute(com.mapbox.navigation.core.fasterroute.NewFasterRoute newFasterRoute);
+    method @UiThread public void destroy();
+    method public void registerNewFasterRouteObserver(com.mapbox.navigation.core.fasterroute.NewFasterRouteObserver observer);
+    method public void unregisterNewFasterRouteObserver(com.mapbox.navigation.core.fasterroute.NewFasterRouteObserver observer);
+  }
+
+  @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public final class NewFasterRoute {
+    ctor public NewFasterRoute(com.mapbox.navigation.base.route.NavigationRoute fasterRoute, double fasterThanPrimaryRouteBy, int alternativeId);
+    method public com.mapbox.navigation.base.route.NavigationRoute component1();
+    method public double component2();
+    method public int component3();
+    method public com.mapbox.navigation.core.fasterroute.NewFasterRoute copy(com.mapbox.navigation.base.route.NavigationRoute fasterRoute, double fasterThanPrimaryRouteBy, int alternativeId);
+    method public int getAlternativeId();
+    method public com.mapbox.navigation.base.route.NavigationRoute getFasterRoute();
+    method public double getFasterThanPrimaryRouteBy();
+    property public final int alternativeId;
+    property public final com.mapbox.navigation.base.route.NavigationRoute fasterRoute;
+    property public final double fasterThanPrimaryRouteBy;
+  }
+
+  @com.mapbox.navigation.base.ExperimentalPreviewMapboxNavigationAPI public fun interface NewFasterRouteObserver {
+    method public void onNewFasterRouteFound(com.mapbox.navigation.core.fasterroute.NewFasterRoute newFasterRoute);
+  }
+
+  public final class SimilarRoutesKt {
+  }
```

Comment:
who knows how to get rid of this?

Reply:
If I'm not mistaken it's a metalava issue and we leave it as is.